ArduCopter/Log.cpp

```diff
@@ -403,7 +407,7 @@ const struct LogStructure Copter::log_structure[] = {
       "DFLT",  "QBf",         "TimeUS,Id,Value", "s--", "F--" },
 #if FRAME_CONFIG == HELI_FRAME
     { LOG_HELI_MSG, sizeof(log_Heli),
-      "HELI",  "Qff",         "TimeUS,DRRPM,ERRPM", "s--", "F--" },
+      "HELI",  "Qffff",        "TimeUS,DRRPM,ERRPM,GOV,THROT", "s----", "F----" },
```

Comment:
No big deal of course but most logging column names are a mix of lower and upper case

Reply:
I will fix this, as well.